list.yaml

```diff
@@ -190,6 +190,6 @@
 - title: Escape the Room
   desc: Have people or admins be able to create rooms that can be solved by people through IRC-like chat
   details:
-    -Make it so that it's necessary for N people to comment in certain ways to solve the room
-    -Make rooms time based so they will come back at certain times or build their own tools to solve rooms
-    -Allow advertisers to build their own rooms and have users solve puzzles based around rooms to get rewards/discounts
+    - Make it so that it's necessary for N people to comment in certain ways to solve the room
```

Comment:
Too hard, just bootstrap it with dildos

Jonathan Ostrander
(516) 640-2963
jonathanost@gmail.com
On Nov 3, 2015 2:23 PM, "Sharang Phadke" <notifications@github.com> wrote:

> In list.yaml
> <https://github.com/SharangP/ideas/pull/3#discussion_r43794017>:
>
> > @@ -190,6 +190,6 @@
> >  - title: Escape the Room
> >    desc: Have people or admins be able to create rooms that can be solved by people through IRC-like chat
> >    details:
> > -    -Make it so that it's necessary for N people to comment in certain ways to solve the room
> > -    -Make rooms time based so they will come back at certain times or build their own tools to solve rooms
> > -    -Allow advertisers to build their own rooms and have users solve puzzles based around rooms to get rewards/discounts
> > +    - Make it so that it's necessary for N people to comment in certain ways to solve the room
>
> @OstrichProjects <https://github.com/OstrichProjects> i already fixed
> this pull master and rebase your branch, or just undo these changes before
> pulling master into your branch:
> ```git checkout your_branch
> git fetch && git rebase origin/master
>
> then fix your merge conflicts
> `git add list.yaml`
> then `git rebase --continue`
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SharangP/ideas/pull/3/files#r43794017>.
>